.azure-pipelines/gpu-tests.yml

```diff
@@ -104,10 +104,6 @@ jobs:
         testRunTitle: '$(Agent.OS) - $(Build.BuildNumber)[$(Agent.JobName)] - Python $(python.version)'
       condition: succeededOrFailed()
 
-    - bash: |
```

Comment:
I think we still shall run some small benchmark or you think the small is too small so it does not make sense?

Reply:
Updated!